projects/plugins/jetpack/_inc/client/product-descriptions/product-description/test/fixtures.js

```diff
@@ -19,9 +19,14 @@ export function buildInitialState() {
 						features: [
 							'Real-time cloud backups',
 							'10GB of backup storage',
-							'30-day archive & activity log',
+							'30-day archive & activity log*',
 							'One-click restores',
 						],
+						disclaimer: {
+							text: '* Subject to your usage and storage limit.',
+							link_text: 'Learn more',
```

Comment:
Does this text here need to be flagged for translation?

Reply:
This is just the test fixture so I don't think we have to worry about the translations here (correct me if I am wrong 😄 )

The text being rendered can be found [here](https://github.com/Automattic/jetpack/pull/25265/files/5aa9c9c648926dd8687062ed6dd1b3094752b2f1#diff-028fe0b6a09c6ed2b0a61f21d8db4d21cf8ca35c8904f94f78503619455fe31fR6734) and is flagged for translation 😄

Reply:
Duh so it is! ~Monday~ Tuesday brain. Thanks!